packages/constants/src/mcf.ts

```diff
@@ -320,3 +320,117 @@ export const JOB_APPLICATION_STATUSES = {
   [JOB_APPLICATION_STATUS.RECEIVED]: 'Received',
   [JOB_APPLICATION_STATUS.WITHDRAWN]: 'Withdrawn',
 };
+
+/*************************************
+ * SSECE EQA List (Education)
+ * Extracted from https://www.singstat.gov.sg/-/media/files/standards_and_classifications/educational_classification/ssec2015-report.pdf
+ *************************************/
+
+const SSEC_EQA_LIST = [
```

Comment:
can we have the interface for `SSEC`?

and maybe `actualCode` rename to `code`?